tbx/static_src/javascript/components/carbon-impact.js

```diff
@@ -0,0 +1,27 @@
+class CarbonImpact {
```

Comment:
My understanding was that we are not going with automatically calculating the page load, as in itself it has a detrimental effect on CO2 usage. The text at mobile should be the same as at desktop.